frontend/simple/js/events.js

```diff
@@ -21,11 +23,12 @@ export class GroupContract extends Events.HashableGroup {
       },
       HashableGroupPayment (state, {data}) { state.payments.push(data) },
       HashableGroupProposal (state, {data, hash}) {
-        // TODO: this should be data instead of ...data to avoid conflict with neighboring properties
-        // TODO: convert to votes instead of for/against for future-proofing
```

Comment:
Why were these TODOs deleted?

Reply:
This probably happened in merging. I'll put them back